project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/model/exception/ExceptionTransformer.kt

```diff
@@ -0,0 +1,20 @@
+package org.plan.research.minimization.plugin.model.exception
+
+import arrow.core.Option
+import arrow.core.toOption
+import org.plan.research.minimization.plugin.execution.IdeaCompilationException
+import org.plan.research.minimization.plugin.execution.exception.KotlincException
+import org.plan.research.minimization.plugin.execution.exception.KotlincException.*
+
+interface ExceptionTransformation {
```

Comment:
It will be better to add `IJDDContext` for each function to provide context for transformation, e.g. `PathRelativizationTransformer` will look much better because we will know the project's path and other info naturally, without strange solutions